pkgs/development/python-modules/torchvision/default.nix

```diff
@@ -34,17 +45,18 @@ in buildPythonPackage rec {
   };
 
   nativeBuildInputs = [ libpng ninja which ]
-    ++ lib.optionals cudaSupport [ cudatoolkit_joined ];
-
-  TORCHVISION_INCLUDE = "${libjpeg_turbo.dev}/include/";
-  TORCHVISION_LIBRARY = "${libjpeg_turbo}/lib/";
+    ++ lib.optionals cudaSupport [ cuda-redist ];
 
-  buildInputs = [ libjpeg_turbo libpng ]
-    ++ lib.optionals cudaSupport [ cudnn ];
+  buildInputs = [ libjpeg_turbo libpng ];
 
   propagatedBuildInputs = [ numpy pillow torch scipy ];
 
-  preBuild = lib.optionalString cudaSupport ''
+  preConfigure = lib.optionalString cudaSupport ''
+    export CC=${cudatoolkit.cc}/bin/cc
```

Comment:
Actually, can we also leave a comment here, for further reference? That we're basically ignoring the toolchain that comes from `stdenv` until we've figured out a better way to substitute one

Reply:
@SomeoneSerge remind me -- since the merge of https://github.com/NixOS/nixpkgs/pull/218265, do I not need to specify these any more?